src/content/en/updates/2017/12/device-memory.md

```diff
@@ -0,0 +1,265 @@
+a [scope](https://support.google.com/analytics/answer/2709828) of "session"
+(since the value will not change during the course of a user's browsing
+session):
+
+<figure>
+  <img src="/web/updates/images/2017/12/device-memory-custom-dimension.png"
+       alt="Creating a Device Memory custom dimensions in Google Analytics"
+       width="290"
+       height="328" />
+</figure>
+
+
+Next update your tracking code. Here's an example of what it might look like.
+Note that for browsers that don't support the Device Memory API, the dimension
+value will be "(not set)".
+
+```
+// Create the tracker from your tracking ID.
+// Replace "UA-XXXXX-Y" with your Google Analytics tracking ID.
+ga('create', 'UA-XXXXX-Y', 'auto');
+
+// Set the device memory value as a custom dimension on the tracker.
+// This will ensure it gets sent with all future data to Google Analytics.
+// Note: replace "XX" with the index of the custom dimension you created
+// in the Google Analytics admin.
+ga('set', 'dimensionXX', navigator.deviceMemory || '(not set)');
+
+// Do any other other custom setup you want...
+
+// Send the initial pageview.
+ga('send', 'pageview');
+```
+
+### Reporting on Device Memory data
+
+Once the device memory dimension is
+[`set`](/analytics/devguides/collection/analyticsjs
+/accessing-trackers) on the tracker object, all data you send to Google
+Analytics will include this value. This will allow you to break down any metric
+you want (e.g. [page load
+times](https://support.google.com/analytics/answer/1205784), [goal completion
+rate](https://support.google.com/analytics/answer/1012040), etc.) by device
+memory, to see if there are any correlations.
+
+Since device memory is a custom dimension rather than a built-in dimension, you
+won't see it in any of the standard reports. To access this data you'll have to
+[create a custom report](https://support.google.com/analytics/answer/1151300).
+For example, the configuration for a custom report that compares load times by
+device memory might look like this:
+
+<figure>
+  <img src="/web/updates/images/2017/12/device-memory-custom-report.png"
+       alt="Creating a Device Memory custom report in Google Analytics" />
+</figure>
+
+And the report it generates might looks like this:
+
+<figure>
+  <img src="/web/updates/images/2017/12/device-memory-results.png"
+       alt="Device Memory report" />
+</figure>
+
+Once you're collecting device memory data and have a baseline for how users are
+experiencing your application across all ranges of the memory spectrum, you can
+experiment with serving different resources to different users (using the
+techniques described in the section above). Afterwards you'll be able to look at
+the results and see if they've improved.
+
+
+## Wrapping up
+
+This post outlines how to use the Device Memory API to tailor your application
+to the capabilities of your users devices, and it shows how to measure how these
```

Comment:
users -> user' (possessive case of a plural noun)

Reply:
Done. (BTW, I think you means "users'", right?)